src/HaroohiePals.MarioKartToolbox/Gui/View/PropertyGrid/MapObjectIdEditor.cs

```diff
@@ -59,8 +59,9 @@ public bool Draw(string label, PropertyEditorContext context)
 
         if (_selectorWindow.Confirm)
         {
-            context.TempActions.Add(
-                new SetPropertyAction(context.Property.SourceObjects, context.Property.Name, _selectorWindow.MapObjectId));
+            bool TTVisible = _mobjDatabase.GetById(_selectorWindow.MapObjectId).IsTimeTrialVisible;
```

Comment:
Variables should use camel case notation, I suggest naming it ``isTimeTrialVisible``.